### PR TITLE
Seeking the Truth + misc.

### PR DESCRIPTION
**Pull request by mplain**

### CODE REVIEW

File: json/Card/fight-on.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+[
+    {
+        "clan": "crab",
+        "cost": 1,
+        "deck_limit": 3,
+        "element": null,
+        "fate": null,
+        "glory": null,
+        "honor": null,
+        "id": "fight-on",
```
Comment (Alsciende) [inline json/Card/fight-on.json]: I don't agree with adding this card to the database. We don't have all the required information (influence_cost).

File: json/Card/seeking-the-truth.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+[
+    {
+        "clan": "neutral",
+        "cost": null,
+        "deck_limit": 1,
+        "element": "water",
```
Comment (Alsciende) [inline json/Card/seeking-the-truth.json]: is that a 100% sure guess?

Comment (mplain) [reply json/Card/seeking-the-truth.json]: Seems to be a cycle, each pack is themed with an element, has a province restricted to a role, province has the same role restriction and element affiliation for Air and Earth at least.
[Air province](https://images-cdn.fantasyflightgames.com/filer_public/75/a6/75a6ef57-7703-48df-9ebe-390c437b99f8/l5c09_cardfan_a1.png)
[Earth province](https://images-cdn.fantasyflightgames.com/filer_public/04/0f/040f1b11-12f0-4d59-87bb-5b2d375c2a8f/l5c10_cardfan_a1.png)

File: json/Card/fight-on.json (hunk 1 above)
Comment (mplain) [reply json/Card/fight-on.json]: As you say.

Comment (Alsciende) [reply json/Card/fight-on.json]: You can remove the file and commit

File: json/Card/my-ancestors-strength.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
+[
+    {
+        "clan": "phoenix",
+        "cost": 1,
+        "deck_limit": 3,
+        "element": null,
+        "fate": null,
+        "glory": null,
+        "honor": null,
+        "id": "my-ancestors-strength",
```
Comment (Alsciende) [inline json/Card/my-ancestors-strength.json]: actually that card already exists too, and it's in the file my-ancestor-s-strength.json

Comment (Alsciende) [reply json/Card/my-ancestors-strength.json]: so you should remove this file

**Pull request merged**